definitions/private-law/json/CaseEvent/EditReturnedOrder/CaseEvent-nonprod.json

```diff
@@ -0,0 +1,21 @@
+[
+  {
+    "LiveFrom": "28/12/2023",
+    "CaseTypeID": "PRLAPPS",
+    "ID": "editReturnedOrder",
+    "Name": "Edit a returned order",
+    "Description": "Event to edit order returned by Judge or manager",
+    "DisplayOrder": 50,
+    "PreConditionState(s)": "CASE_ISSUED;JUDICIAL_REVIEW;PREPARE_FOR_HEARING_CONDUCT_HEARING;DECISION_OUTCOME",
+    "PostConditionState": "*",
+    "SecurityClassification": "Public",
+    "ShowSummary": "Y",
+    "ShowEventNotes": "N",
+    "CallBackURLAboutToStartEvent": "${CCD_DEF_COS_URL}/edit-returned-order/about-to-start",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/judge-or-admin-edit-approve/about-to-submit",
```

Comment:
judge-or admin?

Reply:
Its an existing end point which is used for edit and serve and edit and approve events.